tests/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM golang:1.15-alpine AS builder
+
+ENV GO111MODULE=on \
+    CGO_ENABLED=0 \
+    GOOS=linux
+
+WORKDIR /build
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
+
+COPY . .
+RUN cd e2e && go test -c -o e2e-tests 
+
+FROM alpine:3.13
+
+RUN apk update && apk add --no-cache --upgrade curl python3 py3-pip
+RUN pip install awscli
+
+RUN apk --purge -v del py-pip
+RUN rm /var/cache/apk/*
+
+RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.15.1/bin/linux/amd64/kubectl
```

Comment:
To keep up to date with latest, can we use:
```
curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
```
Outlined here: https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/

Unless we need 1.15.1 for a specific reason.